go.mod

```diff
@@ -4,8 +4,9 @@ go 1.16
 
 require (
 	github.com/gin-gonic/gin v1.6.3
```

Comment:
looks like this line should be removed? we have no gin-gonic use anymore? what am I missing?

Reply:
yes

Reply:
I ran go mod tidy is removed